readme.md

```diff
@@ -150,3 +150,48 @@ The best time for the whole team to start is at the "ideation" or "Design Sprint
 stage so that no time is wasted on this on-boarding later on.
 
 Further reading: https://en.wikipedia.org/wiki/Brooks%27s_law
+
+## When to create a new issue when dealing with bugs 🐛  and enhancements :ribbon:
+
+An existing issue is put into `please-test` as the team have finished working on
+it and all of the acceptance criteria have been fulfilled. You, the product
+owner test the issue and on reflection you decide that you think the designs
+would look better with a couple of tweaks - maybe removing the bold on the
+title, making the logo a bit bigger and changing the copy by a few words. What
+do you do in this situation?
+
+These changes might relate to the changes made in the issue but if they are
+fundamentally not what you asked for in the first place or not what was shown on
+the mock-up then they belong in a new issue **NOT** as a 'bug' or as a request
+at the bottom of the existing issue. Small as they seem, these issues are
+enhancements and new scope and they should be dealt with separately.
+
+### Why?
+These kinds of small changes add up, especially if you change your mind a couple
+of times or if there are a few across multiple issues. Not only do the github
+issues become muddied as their title may no longer reflect the content inside
+but also it becomes more complicated for people to follow the chains of
+conversations that have evolved. Also, the extra changes will impact the
+developer's initial time estimate which is damaging for their accuracy when
+trying to estimate how long issues will take. Imagine if these small changes
+prevented another issue in the sprint from being able to be completed which had
+you put the changes side by side, you would have prioritised the existing issue
+over these small tweaks. It's fine if you do want to prioritise getting these
+changes made but do it in a new issue where the time and importance can be seen
+by everyone and changes can be made to remove another issue from the sprint if
+prioritising this new one has to push another one out.
+
+### So what changes count as bugs :bug: then?
+Bugs are places where the acceptance criteria has not been met :negative_squared_cross_mark: :
+
+ie. you were meant to change the font size across all titles on this page but
+you missed out the last title - please can you change it?
+
+**OR**
+
+Places where the changes made in an issue have broken something else that
```

Comment:
Happy to have this in here but it could get messy - if the cancel button colour is changed on another page when you're only supposed to change the cancel button colour on this page, do you open a new bug issue for the other page or do you add it to the current issue which is not relevant to the other page?